src/Physics/InstantaneousTimeMirrorParameters.h

```diff
@@ -0,0 +1,32 @@
+#ifndef SEISSOL_ITMCONFIG_H
+#define SEISSOL_ITMCONFIG_H
+
+#include <memory>
+#include <yaml-cpp/yaml.h>
+
+namespace seissol::initializers::ITM {
+class ITMParameters {
```

Comment:
Probably I had asked somewhere else already, but: do we need these parameters as well as the reading in the `InputParameters.{c|h}pp` files?

Reply:
No, corrected and pushed.